OptimizelySDK/OptimizelyTests/DemoSwiftApp/Samples/SamplesForAPI.swift

```diff
@@ -0,0 +1,105 @@
+//
+/****************************************************************************
```

Comment:
probably don't need to have sample apis here.

Reply:
I was planning to remove the entire DemoSwiftApp folder from OptimizelyTests as soon as PR #236 gets reviewed. PR #236 has to do with UI Tests for backgrounding, and involves using the DemoSwiftApp's `bundleIdentifier` to access the demo app from a different project file.